Source/UI/Activation/Scenes/FakePasswordViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
```

Comment:
What is this class good for? Should it really be committed in the project? I assume this is just for development / debug purposes... (?)

Reply:
Well, the project is still in development and that controller is just temporary solution. Don't panic!